.eslintrc

```diff
@@ -14,6 +14,7 @@
   "rules": {
     "arrow-parens": ["error", "always"],
     "import/no-unresolved": "off",
+    "no-underscore-dangle": "off",
```

Comment:
Yep, looks like eslint only catches assignments like that though - otherwise we'd have hit that alot. I only found 3 no-underscore-dangle disabling comments.